specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/examples/diskExamples/Disk_CreateOrUpdate_BurstingEnabled.json

```diff
@@ -36,6 +36,7 @@
           },
           "diskSizeGB": 1024,
           "burstingEnabled": true,
+          "burstingEnabledTime": "2022-06-28T04:41:35.079872+00:00",
```

Comment:
This property cannot be set by the client. It is only returned in the response to a request that enables bursting and by Get Disk

Reply:
Removed